### PR TITLE
Update lazer download link in `Help centre`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Help_centre/Upgrading_to_lazer/en.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ The following is a comprehensive list of the **current state** of lazer in compa
 
 So you've decided you want to give lazer a shot? Great!
 
-You can find it for download [here](https://github.com/ppy/osu#running-osu). In the near future, you will be able to switch to lazer from stable (from the `Release stream` setting) and find the download link on the osu! website.
+You can find it for download [here](https://osu.ppy.sh/home/download). In the near future, you will be able to switch to lazer from stable (from the `Release stream` setting) and find the download link on the osu! website.
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/en.md:114]:
```suggestion
You can find it for download [here](https://osu.ppy.sh/home/download). In the near future, you will be able to switch to lazer from stable (from the `Release stream` setting).
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Upgrading_to_lazer/en.md (hunk 1 above)
Comment (agatemosu) [reply wiki/Help_centre/Upgrading_to_lazer/en.md:114]: how fast

**Review by Walavouchey — APPROVED** [review]

**Review by The-Last-Cookie — APPROVED** [review]
German is okay :+1:

**Comment by agatemosu** [comment]
@CloneWith plz review

**Comment by CloneWith** [comment]
Chinese translation is ok ✅

**Review by CloneWith — APPROVED** [review]

**Comment by Sclafus** [comment]
Italian looks good

**Review by cl8n — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**